### DISCUSSION TITLE
Why does a $ show up in my git_status module only in a certain git repository?

### DISCUSSION THREAD

**Discussion by poperigby** [starship/starship#1864] [Q&A] [answered]
It sits inside the square brackets, like this `[$]`, and if there are modifications to the repo, it will prepend itself to the usual symbol, like this `[$!]`.

**Comment by ghost** [comment, marked as answer]
The dollar symbol means that something is currently stashed. A explanation for all git symbols can be found [here](https://starship.rs/config/#git-state).

**Reply by poperigby** [reply]
Oh, that makes sense. Thank you.
